### ISSUE TITLE
Applications,Launcher,Workspace icons disapear from dock.

### ISSUE THREAD

**Issue by bflanagin** [pop-os/cosmic-dock#17]
**How did you upgrade to 21.04? (Fresh install / Upgrade)**
 Upgrade from 20.10

 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
Ubuntu-Dock 

```
pop-cosmic:
  Installed: 0.1.0~1622730049~21.04~7b17096
  Candidate: 0.1.0~1622730049~21.04~7b17096
  Version table:
 *** 0.1.0~1622730049~21.04~7b17096 1001
       1001 http://ppa.launchpad.net/system76/pop/ubuntu hirsute/main amd64 Packages
       1001 http://ppa.launchpad.net/system76/pop/ubuntu hirsute/main i386 Packages
       1001 http://ppa.launchpad.net/system76/proposed/ubuntu hirsute/main amd64 Packages
       1001 http://ppa.launchpad.net/system76/proposed/ubuntu hirsute/main i386 Packages
        100 /var/lib/dpkg/status
```

**Issue/Bug Description:**

Applications,Workspaces, and Launcher icons were removed with no way to restore them. Another user reports that these three icons weren't present to begin with. https://github.com/pop-os/beta/issues/151#issuecomment-854888031

Trying to recreate the issue on a test account.

**Comment by bflanagin** [comment]
This seems to only happen to pre-beta users.

## Workaround

Once the icons have been removed from the dash you can add them back by using the applications view (not the launcher) and manually adding them to your favorites.

However I don't know if this will work once the add/remove settings are implemented.

**Comment by lbkNhubert** [comment]
I may be missing something, but I do not see these items in my applications view to be able to add them to the favorites. I had upgraded from 20.10 to 21.04 beta. Initially the icons were present, at one point (after a software update, if memory serves) they disappeared and I have not been able to restore them. Thank you for your help!

Edit - I added them to the favorites with dconf Editor, and looking at the .desktop file see that they are under Utilities. I do see them there now, but I thought that I had checked all of the folders in applications. Maybe I missed them previously. At any rate, I have them back in the menu now.
